packages/calling/src/CallHistory/CallHistory.test.ts

```diff
@@ -247,4 +251,117 @@ describe('Call history tests', () => {
       );
     });
   });
+
+  describe('fetchUCMLinesData test', () => {
+    it('verify successful UCM lines API case', async () => {
+      const ucmLinesAPIPayload = <WebexRequestPayload>(<unknown>MOCK_LINES_API_CALL_RESPONSE);
+
+      webex.request.mockResolvedValue(ucmLinesAPIPayload);
+      // Since fetchUCMLinesData is a private method, TypeScript restricts direct access to it.
+      // To bypass this restriction, we are using 'as any' to access and invoke the method for testing purposes.
+      const response = await (callHistory as any).fetchUCMLinesData();
+
+      expect(response.statusCode).toBe(200);
+      expect(response.message).toBe('SUCCESS');
+    });
+
+    it('verify bad request failed UCM lines API case', async () => {
+      const failurePayload = {
+        statusCode: 400,
+      };
+      const ucmLinesAPIPayload = <WebexRequestPayload>(<unknown>failurePayload);
+
+      webex.request.mockRejectedValue(ucmLinesAPIPayload);
+      // Since fetchUCMLinesData is a private method, TypeScript restricts direct access to it.
+      // To bypass this restriction, we are using 'as any' to access and invoke the method for testing purposes.
+      const response = await (callHistory as any).fetchUCMLinesData();
```

Comment:
Does this not work?
```suggestion
      const response = await callHistory['fetchUCMLinesData']();
```

Reply:
Thank you for the suggestion. Taken care of it.